arkouda/dataframe.py

```diff
@@ -104,14 +106,48 @@ class DataFrameGroupBy:
         If True the grouped values of the aggregation keys will be treated as an index.
     """
 
-    def __init__(self, gb, df, gb_key_names=None, as_index=True):
+    def __init__(self, gb, df, gb_key_names=None, as_index=True, dropna=True):
+
         self.gb = gb
         self.df = df
         self.gb_key_names = gb_key_names
         self.as_index = as_index
         for attr in ["nkeys", "permutation", "unique_keys", "segments"]:
             setattr(self, attr, getattr(gb, attr))
 
+        self.dropna = dropna
```

Comment:
You don't have to pass in `dropna` if you don't want to b/c `self.gb.dropn`a is already set.

Reply:
oh yah that's a good point!